src/index.html

```diff
@@ -11,7 +11,166 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="form-group">
+        <legend>Personal information</legend>
+        <div class="form-fild">
+          <label for="Surname">Surname:</label>
+          <input
+               type="text"
+               name="Surname"
+               autocomplete="off" 
+               id="Surname"
+          >
```

Comment:
It should be 2 spaces not 5. Fix all over page
```suggestion
          <input
            type="text"
            name="Surname"
            autocomplete="off" 
            id="Surname"
          >
```

Reply:
https://DmitriyChernega.github.io/layout_html-form/

DmitriyChernega.github.io/layout_html-form/report/html_report/